src/test/java/com/connexta/ingest/IngestITests.java

```diff
@@ -244,6 +244,7 @@ public void testSuccessfulIngestRequest() throws Exception {
         .header(
             "Accept-Version",
             "0.5.0") // TODO inject this like we do for the transformApiVersion in TransformClient
+        .header("Last-Modified", "2017-06-11T14:32:28Z")
```

Comment:
✏️Unlike `Accept-Version`, `Last-Modified` is a defined header name (https://tools.ietf.org/html/rfc7232#section-2.2). Can a library be used to set it here instead of manually setting the header name to `"Last-Modified"`? Would this work?
```java
.headers(httpHeaders -> httpHeaders.setLastModified(...))
```

Reply:
![image](https://user-images.githubusercontent.com/11355332/66523086-e12e7d80-eaa3-11e9-90b8-b6f84faba90f.png)
I tried using the ZonedDateTime implementation and found it caused the test to fail because the resulting date was not in the expected format. Additionally the code didn't seem to be more concise or readable (opinion for sure). 
`.headers(httpHeaders -> httpHeaders.setLastModified(ZonedDateTime.parse(dates)))`
Are there other advantages to pursing the library approach instead of setting it directly using the header name?

Reply:
I'm fine using the direct header name. Thanks for looking into that:+1:

Reply:
I'm fine using the header name directly. I'm worried that Replication Team will run unto an issue when they try to send us a request using these spring libraries to set the timestamp, but we can address that if/when it happens. Thanks for looking into that👍